README.md

```diff
@@ -14,6 +14,34 @@
 
 This repo contains Jupyter Notebooks useful for several aspects of Blue Team work.
 
+## Installation
+
+### Linux
+You need Jupyter Notebooks! On linux, run:
+```
+$ sudo apt-get install jupyter
+```
+Clone and enter the directory:
+```
+$ git clone https://github.com/mttaggart/blue-jupyter.git && cd blue-jupyter
+```
+If you don't have it already, install `poetry`:
```

Comment:
Once we have this thing in a stable state, the plan is to also create a proper `requirements.txt`

Reply:
Apt-get install replaced with pip